src/styles/README.md

```diff
@@ -1,4 +1,36 @@
 # NYPL Design System Styles
+
+For an overview of the entire Design System, check out the Design System [README](https://github.com/NYPL/nypl-design-system/blob/development/README.md).
+
 ## How to Use
+The Design System Styles provides the full and uncompiled scss files. The product is responsible to import the styles into its project. To import the package, run
+
+```
+  npm install --save @nypl/design-system-styles
+```
+
+Configurations need to be updated to support importing files. Check out the [storybook config](https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-react/webpack.config.js) as a full example.
```

Comment:
change this to [react storybook config](https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-react/webpack.config.js) or [twig storybook config]((https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-html/webpack.config.js)